examples/src/clipping.ts

```diff
@@ -23,7 +23,7 @@ async function main() {
   renderer.setSize(window.innerWidth, window.innerHeight);
   document.body.appendChild(renderer.domElement);
 
-  const revealManager: reveal.RevealManager = createRenderManager(
+  const revealManager: reveal.RenderManager = createRenderManager(
```

Comment:
I don't think we should expose RenderManager to the user - let the user deal with RevealManager and hide away other managers as much as possible.

Reply:
Not possible in the current implementation. LocalHostRevealManager and RevealManager are different in signature, meaning you can't save LocalHostRevealManager to a variable of type RevealManager.

Reply:
Both are `RevealManagerBase`?

Reply:
Yes, but as you remember it needs a type and it would have to be a RevealManagerBase<any> which we agreed that we didn't like.